composer/utils/fx_utils.py

```diff
@@ -111,6 +113,40 @@ def replace_op(gm: GraphModule, src_ops: Union[Callable, str, List[Union[Callabl
     return gm
 
 
+class BlockStochasticModule(nn.Module):
+    """A convenience class that stochastically executes the provided non-residual path.
+
+    Args:
+        lhs (GraphModule): Operators in the non-residual path of a residual block.
```

Comment:
nit: `lhs` and `rhs` are not descriptive -- consider using `main` vs `residual` as the arguments?

Reply:
How about residual and critical? critical path because it has more operations.

Reply:
Renamed to residual and main